integrated_channels/sap_success_factors/client.py

```diff
@@ -46,6 +46,9 @@ def __init__(self, enterprise_configuration):
         self.global_sap_config = apps.get_model('sap_success_factors', 'SAPSuccessFactorsGlobalConfiguration').current()
         self.session = None
         self.expires_at = None
+        self.IntegratedChannelAPIRequestLogs = apps.get_model(
+            "integrated_channel", "IntegratedChannelAPIRequestLogs"
+        )
```

Comment:
I think this can be removed? 🤔

Reply:
@hamzawaleed01 updated to use it to store api call data with no payload, to keep consistent wih other channels.